Model/Config/Structure/Data.php

```diff
@@ -61,9 +57,8 @@ public function get($path = null, $default = null)
         return $data;
     }
 
-    protected function isSectionVisible($section)
+    private function isSectionVisible($section): bool
     {
-        $scope = "showIn" . ucfirst($this->scope);
-        return isset($section[$scope]) && $section[$scope];
+        return $section['showIn' . ucfirst($this->_configScope->getCurrentScope())] ?? false;
```

Comment:
Vendor titles not appearing under Store > Configuration > Extensions. Possibly this change caused this problem.

Reply:
I don't get the `isSectionVisible` method purpose. I've removed it, and sections not visible in specific scope are not when they should. What is the intent of this method?

Reply:
@thomas-kl1 This method checks if the section is allowed under the currently selected configuration scope global/website/store/store view. so that we add "original tab title"/vendor_title to a visible section only.

This method fixes a  bug that vendor title doesn't appear for some extension vendors having the first section of their tab disallowed for the default scope.

For example, try installing this extension https://www.wyomind.com/magento2/security-enhancement-magento.html and notice Store > Configuration > Extension area. You will not see orange "Wyomind" title for their settings.

Reply:
I've installed the module and I don't see any issue. The orange title is visible in default, not in website and store, which is normal behavior (there is no available settings in these scopes).
Visibility is already managed Magento core, so to me it's more a duplicate here. Did I missed something in the test?

Reply:
Something might have missed during the test. I will check and get back to you by today or tomorrow.

Reply:
In the meantime, let's not keep the bugfix pending. 

May you please create a separate pull request just for the bug fix or may I just add it from my side and make a release directly. So that it can be merged and we discuss other things after that.

Reply:
I'll do a separate PR

Reply:
Awesome! Looking forward to it.

Reply:
> I've installed the module and I don't see any issue. The orange title is visible in default, not in website and store, which is normal behavior (there is no available settings in these scopes).
> Visibility is already managed Magento core, so to me it's more a duplicate here. Did I missed something in the test?

Just checked this. It seems like, they have updated their extension. Please try changing file manually to repeat the issue

vendor/wyomind/core/etc/adminhtml/system.xml

Change showInDefault to 0 from 1 for the section tag

Reply:
> Change showInDefault to 0 from 1 for the section tag

It's definitly because of the third party module bug. I don't think this module should fix the bug of other parties.

Reply:
Yes, it shouldn't. 

But the end-user installing the clean menu extension expects that it should properly move and display all the extension settings under Store > Configuration > Extensions

If a title of extension vendor is not displayed there (due to XYZ reasons), then he will not accept if it's due to the bug in the 3rd party extension instead of clean menu because the title is visible correctly prior installing clean menu.

So, if we can handle such scenario's within our extension then that should be the better approach.

I hope this makes sense.

Reply:
With the isSectionVisible in Structure Data, when showInDefault is 1 in Wyomind_Core:
![Screenshot from 2020-06-10 09-41-17](https://user-images.githubusercontent.com/20971693/84240616-98313c00-aafe-11ea-948b-cfbe90278f21.png)

isSectionVisible + showInDefault is 0 in Wyomind_Core
![Screenshot from 2020-06-10 09-42-57](https://user-images.githubusercontent.com/20971693/84240785-cca4f800-aafe-11ea-8e81-2c34572e4999.png)
There is no more vendor title even if CronScheduler of Wyomind is there.

Without isSectionVisible + showInDefault is 0 in Wyomind_Core
![Screenshot from 2020-06-10 09-42-57](https://user-images.githubusercontent.com/20971693/84240785-cca4f800-aafe-11ea-8e81-2c34572e4999.png)

Without isSectionVisible + showInDefault is 1 in Wyomind_Core
![Screenshot from 2020-06-10 09-41-17](https://user-images.githubusercontent.com/20971693/84240616-98313c00-aafe-11ea-948b-cfbe90278f21.png)

I don't see any difference by using isSectionVisible (not modified)

So in my opinion there is an other bug, the tab is only depending on the first section visibility.

So I've updated the PR with proper fix

Reply:
Make sense. I am testing the changes and get back to you by today.